<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NonuniformityCorrectionDialog</class>
 <widget class="QDialog" name="NonuniformityCorrectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nonuniformity Correction</string>
  </property>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nonuniformity Correction (BFC)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="runStepButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Step &gt;</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>331</width>
     <height>16</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="stepNameText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Initialization</string>
   </property>
  </widget>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QPushButton" name="previousStepButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt; Back</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="nextTool">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Next Stage: Tissue Classification</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="iterativeModeCheckBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Iterative mode</string>
   </property>
  </widget>
  <widget class="QLabel" name="histogramRadiusLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Histogram radius</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="histogramRadiusEditBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="sampleSpacingLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sample spacing</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sampleSpacingEditBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="controlPointSpacingEditBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="controlPointSpacingLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Control point spacing</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="splineStiffnessEditBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="splineStiffnessLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Spline stiffness</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="cuboidRadioButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>82</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>cuboid</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="ellipsoidRadioButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>210</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>ellipsoid</string>
   </property>
  </widget>
  <widget class="QLabel" name="roiShapeLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>ROI Shape</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="biasRangeUpperEditBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>240</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="biasRangeLowerEditBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="biasEstimateRangeLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Bias estimate range</string>
   </property>
  </widget>
  <widget class="QPushButton" name="lowRangeButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>low</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mediumRangeButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>medium</string>
   </property>
  </widget>
  <widget class="QPushButton" name="highRangeButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>high</string>
   </property>
  </widget>
  <widget class="QLabel" name="autosaveLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QPushButton" name="restoreFromPreviousSessionButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>191</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Restore From Previous...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
